README.md

```diff
@@ -4,9 +4,9 @@ This repository contains templates for all *aquality automation* family librarie
 
 ### Structure
 
-- [**java/**](./java) (will be ready soon)
+- [**java/**](./java)
   - [**aquality-selenium-template**](./java/aquality-selenium-template)
-  - [**aquality-appium-mobile-template**](./java/aquality-appium-mobile-template)
+  - [**aquality-appium-mobile-template**](./java/aquality-appium-mobile-template) (will be ready soon)
 - [**dotnet/**](./dotnet) (will be ready soon)
   - [**aquality-selenium-template**](./dotnet/aquality-selenium-template)
   - [**aquality-appium-mobile-template**](./dotnet/aquality-appium-mobile-template)
```

Comment:
so basically person won't be able to download a single template, it would be necessary to download a whole repository, am I correct?

Reply:
Yes. I beleive it is better to hold templates in one repository.

Reply:
May we use a submodules for this purpose? 
Or alternatively use a different branches for different templates.
Seems we should make it easier to getting started for a new user... I would rather not clone/download a repository with many projects in different languages inside

Reply:
Do you mean maven submodules? Are you sure it is easier? Definitly no. Not sure about branches. From my point of view branches also bad idea.

Reply:
i mean git submodules https://git-scm.com/book/en/v2/Git-Tools-Submodules